Readme/automation.md

```diff
@@ -0,0 +1,155 @@
+  - Select **Hybrid/Native from Apps** in `App type` section. 
+
+ Kobiton automatically generates your desired capabilities into code based on the selected options. Note values of `deviceName`, `platformversion` and `platformName` to prepare next steps.
+
+  ![desiredcaps](./assets/desiredcaps.png) 
+### 2. Configuring automation test script
+In the `automation-script` folder, we have provided a sample scripts for executing automation test on Kobiton devices.
+As mentioned above, the desired capabilities in provided automation test scripts have already been pre-configured. If you have chosen custom desired capabilities, please follow steps below to apply the collected desired capabilities.
+
++ Add there environment variables to TeamCity project:
+  + KOBITON_DEVICE_NAME: Kobiton device name ( `deviceName` in the generated desired capabilities)
+  + KOBITON_DEVICE_PLATFORM_NAME: Kobiton device platform name ( `platformName` in the generated desired capabilities)
+  + KOBITON_DEVICE_PLATFORM_VERSION: Kobiton device platform version ( `platformversion` in the generated desired capabilities)
+
+![env](./assets/env_designCap.png)
+ 
+For example, if desired capabilities for executing automation test on Pixel 2 XL running Android 8.1.0
+```javascript
+var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: 'This is an example for Android app', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<APP_URL>', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         'Pixel 2 XL',
+  platformVersion:    '8.1.0',
+  platformName:       'Android' 
+}
+```
+> More information about automation testing with Kobiton can be found at https://docs.kobiton.com/automation-testing/automation-testing-with-kobiton/.
+### 3. Configuring HockeyApp Webhooks listenner
+Kobiton has provided script for setting up a small, basic server for receiving WebHook notifications from HockeyApp and triggering TeamCity to execute automation test of that application on Kobiton. The script will load settings from a configuration file and start a WebHook listener. This script must be executed on the server mentioned in [Prerequisites](#1-prerequisites).
+
+In order to use the provided script, please make some modifications to the configuration file:
+
+1. Navigate to `/sample/server/` and open `server.rb` file.
+
+2. Replace with your own values:
++ `teamcity_username`: Your TeamCity username
++ `teamcity_password`:  Your TeamCity password
++ `port`: Web server's port number. Default: 3000
++ `build_id`: TeamCity Build configuration ID
+3. Execute the provided script using the below command:
+```bash
+ruby server.rb
+```
+4. In your application configuration in HockeyApp, create Webhooks from server address to your corresponding server address, port.
+>  Follow [HockeyApp Webhooks](https://www.hockeyapp.net/blog/2013/08/06/hockeyapp-webhooks.html) to instruction to create one. 
```

Comment:
for instructions. This line is confusing and ambiguous. Please make it more clear.